src/Fossa/API/Types.hs

```diff
@@ -216,23 +217,69 @@ instance Ord IssueSummaryTarget where
       then comparing istTargetPaths lhs rhs
       else comparing istTargetType lhs rhs
 
+-- This is not yet used, but may need to be in the future.
+-- Reevaluate before making PR.
+data IssueCategory
+  = Security
+  | Compliance
+  | Other Text
+  deriving (Eq, Ord, Show)
+
+instance ToText IssueCategory where
+  toText :: IssueCategory -> Text
+  toText i =
+    case i of
+      Security -> "Security"
+      Compliance -> "Compliance"
+      Other t -> t
+
+instance Pretty IssueCategory where
+  pretty = pretty . toUpper . toText
+
+issueTypeToCategory :: IssueType -> IssueCategory
+issueTypeToCategory =
+  \case
+    IssueAbandonware -> Security
+    IssueDenyListedDep -> Security
+    IssueEmptyPackage -> Security
+    IssueNativeCode -> Security
+    IssueOutdatedDependency -> Security
+    IssueVulnerability -> Security
+    IssuePolicyConflict -> Compliance
+    IssuePolicyFlag -> Compliance
+    IssueUnlicensedAndPublicDep -> Compliance
+    IssueUnlicensedDependency -> Compliance
+    IssueOther t -> Other t
+
+-- These constructors are ordered alphabetically.
+-- This is needed for our `fossa test` output.
 data IssueType
-  = IssuePolicyConflict
+  = IssueAbandonware
+  | IssueDenyListedDep
+  | IssueEmptyPackage
+  | IssueNativeCode
+  | IssueOther Text
+  | IssueOutdatedDependency
+  | IssuePolicyConflict
   | IssuePolicyFlag
-  | IssueVulnerability
+  | IssueUnlicensedAndPublicDep
   | IssueUnlicensedDependency
-  | IssueOutdatedDependency
-  | IssueOther Text
+  | IssueVulnerability
   deriving (Eq, Ord, Show)
 
-renderIssueType :: IssueType -> Text
-renderIssueType = \case
-  IssuePolicyConflict -> "Denied by Policy"
-  IssuePolicyFlag -> "Flagged by Policy"
-  IssueVulnerability -> "Vulnerability"
-  IssueUnlicensedDependency -> "Unlicensed Dependency"
-  IssueOutdatedDependency -> "Outdated Dependency"
-  IssueOther other -> other
+instance Pretty IssueType where
+  pretty = \case
+    IssueAbandonware -> "Abandoned Dependencies"
+    IssueDenyListedDep -> "Denylisted Dependency"
+    IssueNativeCode -> "Native Code Dependency"
+    IssuePolicyConflict -> "Denied by Policy"
+    IssuePolicyFlag -> "Flagged by Policy"
+    IssueVulnerability -> "Vulnerability"
+    IssueUnlicensedDependency -> "Unlicensed Dependency"
+    IssueUnlicensedAndPublicDep -> "Unlicensed and Public Dependency"
```

Comment:
The descriptions I found in core are these.

unlicensed and public:
>  Since this project is public, in order for others to use it, you must specify a license otherwise it stays copyrighted to you.

unlicensed:
> These dependencies contain unlicensed code, which by default is copyrighted to the original author. You may not use these dependencies without first obtaining a license.